controllers/machine_controller_phases_test.go

```diff
@@ -681,13 +681,22 @@ func TestReconcileBootstrap(t *testing.T) {
 			},
 		},
 		{
-			name: "existing machine, bootstrap provider is not ready",
+			name: "existing machine, bootstrap provider is not ready, and ownerref updated",
```

Comment:
Sorry to think of this at the last minute, but could you please add a **new** test case that hard-codes the v1alpha2 API version for a MachineSet owner, just to cover all our bases? Thanks!